### ISSUE TITLE
[COGNITION]: Pagination needs a visual and aural marker of current page

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#5958]
## Description

The pagination being used on some pages is not showing users which page is current, and is not announcing which page is current to screen readers. This could be confusing because there's no clear indication which links are clickable. Screen shot attached below.

## Type of Issue
Template

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

**Definition of done**
* [ ] As a user, I should be able to understand which pagination link is current. The design system component denotes this as a blue circle around the current page: https://design.va.gov/components/pagination
* [ ] As a screen reader user, I want to hear the current page being announced. This can be accomplished with `aria-current="page"`
* [ ] As a keyboard user, I should be able to set focus on every pagination link and understand which one has focus by the yellow halo.

## Environment
* https://www.va.gov/pittsburgh-health-care/stories/page-2/(among others, anywhere we use pagination)

## Possible Fixes (optional)

The pagination links need to be refactored to visually match the design system component. They also need some semantic markup added. I'd like to get away from using `aria-label` because it doesn't scale when page counts get higher. Using `<span>` with the SR-only text is the current preferred method.

```html
<div class="va-pagination">
  <div class="va-pagination-inner">
    <a aria-current="page" href="/pittsburgh-health-care/events/past-events/page-2">
      <span class="vads-u-visibility--screen-reader">Page</span>
      1
    </a>
    <a href="/pittsburgh-health-care/events/past-events/page-2">
      <span class="vads-u-visibility--screen-reader">Page</span>
      2
    </a>
  </div>
  <span class="va-pagination-next">
    <a href="/pittsburgh-health-care/events/past-events/page-2">
      Next
      <span class="vads-u-visibility--screen-reader">page of past events</span>
    </a>
  </span>
</div>
```

## WCAG or Vendor Guidance (optional)

* [Link Purpose (In Context): Understanding SC 2.4.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html)

## Screenshots or Trace Logs

![Screen Shot 2020-02-18 at 2 35 26 PM](https://user-images.githubusercontent.com/934879/74776686-a1e81880-525d-11ea-9766-402081bcd97a.png)

**Comment by mmiddaugh** [comment]
This issue is resolved
Screenshots from an example [VAMC events page](https://www.va.gov/pittsburgh-health-care/events/past-events/)

![image](https://user-images.githubusercontent.com/55411834/110856598-14ff6b00-8275-11eb-8204-d3c602965737.png)
![image](https://user-images.githubusercontent.com/55411834/110856955-83dcc400-8275-11eb-8ff7-3240d77489ef.png)
